### ISSUE TITLE
When Playing Star Wars: Squadrons, OVRDrop stops displaying just a window and shows the entire screen

### ISSUE THREAD

**Issue by cigdangle** [Hotrian/OVRdrop-Public#78]
OVRDrop works just fine, until I launch Star Wars: Squadrons or if I ALT-Tab in and out of the game.  Other games work just fine.

Upon launch, instead of showing just the windowed app, the entire monitor/desktop displays.
[output_log.txt](https://github.com/Hotrian/OVRdrop-Public/files/5379636/output_log.txt)

**Comment by Hotrian** [comment]
That's odd. The log file isn't indicating any reason for that. Is SW:S running in fullscreen?

**Comment by cigdangle** [comment]
Yes; full screen.  It's very odd, and seems to be the only program with the issue.

I'm not certain, but it may have only been occurring since the most recent patch.

**Comment by cigdangle** [comment]
Here is a video of the "weirdness" occurring. 

[https://youtu.be/hpDsNiq8hWc](https://youtu.be/hpDsNiq8hWc)
